contracts/account/library.cairo

```diff
@@ -17,6 +17,11 @@ from starkware.cairo.common.bool import TRUE, FALSE
 
 from contracts.upgrade.Upgradable import _set_implementation
 
+from contracts.utils.calls import (
+    Call,
```

Comment:
I don't think we need to import the `Call` struct here has it is not being used.

Reply:
it's actually used in line 45, we need to convert from the crazy `CallArray` format to the more usable `Calls`

Reply:
Are you sure? I can't find it being used anywhere in the file.

Reply:
you are right, i was looking at the wrong file. I've now removed the import from `library.cairo` and `ArgentAccount.cairo`